Open Judge System/Workers/OJS.Workers.LocalWorkerMonitoring/App.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="EmailServerHost" value="smtp.softuni.com" />
+    <add key="EmailServerPort" value="25" />
+    <add key="EmailServerUsername" value="TestUser" />
+    <add key="EmailServerPassword" value="Test" />
+    <add key="EmailSenderEmail" value="test@test.test" />
+    <add key="EmailSenderDisplayName" value="Test" />
```

Comment:
Are these credentials valid for production?

Reply:
No, the production credentials should not be committed to the repository, but deployed manually only to the server.